src/libraries/System.Linq.Expressions/src/CompatibilitySuppressions.xml

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<!-- https://learn.microsoft.com/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
```

Comment:
```suggestion
<!-- https://learn.microsoft.com/dotnet/fundamentals/package-validation/diagnostic-ids -->
```

FYI - it looks like this was recently fixed with https://github.com/dotnet/sdk/pull/42830. We are just using an out of date version of the tool.

cc @ViktorHofer